### PR TITLE
various fixes 2

### PR DESCRIPTION
**Pull request by Evankhell561**
fixes several of the repository's issues.

someday:
image

### CODE REVIEW

File: code/__defines/items_clothing.dm [hunk 1]
```diff
@@ -101,11 +101,9 @@
 #define slot_r_ear_str      "slot_r_ear"
 #define slot_belt_str       "slot_belt"
 #define slot_shoes_str      "slot_shoes"
-#define slot_head_str      	"slot_head"
```
Comment (NanakoAC) [inline code/__defines/items_clothing.dm:104]: why are these removed? I added them recently and will need them in future

File: code/__defines/misc.dm [hunk 2]
```diff
@@ -188,7 +188,6 @@
 #define CELLS 8								//Amount of cells per row/column in grid
 #define CELLSIZE (world.icon_size/CELLS)	//Size of a cell in pixels
 
-#define WORLD_ICON_SIZE 32
```
Comment (NanakoAC) [inline code/__defines/misc.dm:191]: This is used frequently including later in this file, whats going on?

File: code/__defines/rust_g.dm [hunk 3]
```diff
@@ -16,7 +16,8 @@
 // On Linux, looks in `.`, `$LD_LIBRARY_PATH`, and `~/.byond/bin` for either of
 // `librust_g.so` (preferred) or `rust_g` (old).
 
-/* This comment bypasses grep checks */ /var/__rust_g
```
Comment (NanakoAC) [inline code/__defines/rust_g.dm:19]: this is clearly written the way it is for a reason

File: code/_helpers/check_trajectory.dm [hunk 4]
```diff
@@ -133,7 +133,7 @@
 /obj/item/projectile/test/expire()
 	return
 
-/obj/item/projectile/test/Bump(atom/A as mob|obj|turf|area)
```
Comment (NanakoAC) [inline code/_helpers/check_trajectory.dm:136]: what's this flag about?

File: code/_helpers/icons.dm [hunk 5]
```diff
@@ -107,7 +107,7 @@ AngleToHue(hue)
 	Converts an angle to a hue in the valid range.
 RotateHue(hsv, angle)
 	Takes an HSV or HSVA value and rotates the hue forward through red, green, and blue by an angle from 0 to 360.
-	(Rotating red by 60� produces yellow.) The result is another HSV or HSVA color with the same saturation and value
```
Comment (NanakoAC) [inline code/_helpers/icons.dm:110]: this looked correct before

File: code/_helpers/text.dm [hunk 6]
```diff
@@ -348,11 +348,11 @@ proc/TextPreview(var/string,var/len=40)
 //For generating neat chat tag-images
 //The icon var could be local in the proc, but it's a waste of resources
 //	to always create it and then throw it out.
-/var/icon/text_tag_icons = new('./icons/chattags.dmi')
+var/icon/text_tag_icons = new('./icons/chattags.dmi')
 /proc/create_text_tag(var/tagname, var/tagdesc = tagname, var/client/C = null)
 	if(!(C && C.get_preference_value(/datum/client_preference/chat_tags) == GLOB.PREF_SHOW))
 		return tagdesc
-	return "<IMG src='\ref[text_tag_icons.icon]' class='text_tag' iconstate='[tagname]'" + (tagdesc ? " alt='[tagdesc]'" : "") + ">"
```
Comment (NanakoAC) [inline code/_helpers/text.dm:355]: trading a variable for a hardcoded value? that doesnt seem useful

**Review by NanakoAC — CHANGES_REQUESTED** [review]
Review barely started, and there's a bunch of issues. What exactly is the purpose of this huge shotgun PR ?

**Comment by Evankhell561** [comment]
I'm going to close this and make a smaller pr.

**Pull request closed without merging**